src/session/Session.ts

```diff
@@ -497,3 +498,18 @@ function setupCompletionHandler(
   }
   return { completionHandler, targetElement, properties };
 }
+
+function getConnectionData(token: string): string {
+  if (!token) return "";
+
+  interface Payload {
+    otcd?: string;
+  }
+
+  const payload = jwt_decode(token);
```

Comment:
Do we need a try catch block here (just in case if the token is not a valid one)

Reply:
Great point, done!